vars/main.yml

```
@@ -170,7 +170,6 @@ k3s_experimental_config:
   - setting: selinux
     until: 1.19.4
   - setting: rootless
-  - setting: secrets-encryption
   - setting: agent-token
```

```suggestion
  - setting: secrets-encryption
    until: 1.23.15
  - setting: agent-token
```

Release notes: [v1.23.15+k3s1](https://github.com/k3s-io/k3s/releases/tag/v1.23.15%2Bk3s1) - lowest supported version with secrets-encryption set as GA

---

Oh! I see, sorry it wasn't clear what that `until` was, sure I'll add that!